README.md

```diff
@@ -518,9 +574,6 @@ namespace Some\Namespace;
 
 use Vendor\Namespace\Entity\Value;
 
-/**
- * @var Value $value
- */
 public function setSomething(Value $value);
```

Comment:
is it fine to leave without a result-type hint? if we go to 7.4 and higher, there should be **:self** or something more specific... with such sample we leave it without any info, as a good example.